core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -94,6 +95,21 @@ protected[controller] object RestAPIVersion_v1 {
 }
 
 /**
+  * A singleton object which defines properties needed to instantiate a service for v2
+  * of the REST API.
+  */
+protected[controller] object RestAPIVersion_v2 {
```

Comment:
I was able to share the MetasApi class. RestAPIVersion_v1 and RestAPIVersion_v2 are a little different unless you have suggestions.